config/settings/base.py

```diff
@@ -311,13 +311,29 @@
 API_INSEE_KEY = os.environ["API_INSEE_KEY"]
 API_INSEE_SECRET = os.environ["API_INSEE_SECRET"]
 
-# Pôle emploi.
+# Pôle emploi's Emploi Store Dev aka ESD.
 # https://www.emploi-store-dev.fr/portail-developpeur/catalogueapi
+# Note that the `EMPLOI_STORE` naming below is incorrect,
+# as the Emploi Store is a totally different beast than the ESD.
 API_EMPLOI_STORE_KEY = os.environ["API_EMPLOI_STORE_KEY"]
 API_EMPLOI_STORE_SECRET = os.environ["API_EMPLOI_STORE_SECRET"]
 API_EMPLOI_STORE_AUTH_BASE_URL = "https://entreprise.pole-emploi.fr"
 API_EMPLOI_STORE_BASE_URL = "https://api.emploi-store.fr/partenaire"
 
+# PE Connect aka PEAMU - technically one of ESD's APIs.
```

Comment:
Nouveau commentaire :

```
# PE Connect aka PEAMU - technically one of ESD's APIs.
# PEAM stands for Pôle Emploi Access Management.
# Technically there are two PEAM distinct systems:
# - PEAM "Entreprise", PEAM-E or PEAME for short.
# - PEAM "Utilisateur", PEAM-U or PEAMU for short.
# To avoid confusion between the two when contacting ESD support,
# we get the habit to always explicitely state that we are using PEAM*U*.
```

Reply:
Pour mémoire j'ai pris cette habitude suite à une grosse galère avec le support ESD pour un souci PE Connect LBB. Après m'avoir baladé d'interlocuteur en interlocuteur, je me suis pris un "Ah bon c'est pas PEAM Entreprise que vous utilisez?". Never again. PEAM*U*.